Plugin~/WebRTCPlugin/WebRTCPlugin.cpp

```diff
@@ -1386,12 +1387,17 @@ extern "C"
         UnityAudioTrackSource* source =
             static_cast<UnityAudioTrackSource*>(track->GetSource());
 
-        source->OnData(audio_data,
+        source->PushAudioData(
+            audio_data,
             sample_rate,
             number_of_channels,
             number_of_frames);
-    }
 
+        auto adm = context->GetAudioDevice();
+        if (adm != nullptr) {
+            adm->RegisterSendAudioCallback(source, sample_rate, number_of_channels);
```

Comment:
This operation is called every frame of processing audio.
Can we call this operation once?

Reply:
Got it, how about this? 91c2e83